src/provider/view/components/NotificationView/NotificationView.tsx

```diff
@@ -108,7 +111,9 @@ export class NotificationView extends React.Component<NotificationViewProps, Not
             window.openfin.notifications.addEventListener(
                 'appNotificationsCleared',
                 (payload: {uuid: string}): string => {
-                    const newNotifications: INotification[] = this.state.notifications.filter(notification => notification.uuid !== payload.uuid);
+                    console.log('appNotificationsCleared triggered', payload);
+                    console.log('current notifications', JSON.stringify(this.state.notifications));
```

Comment:
Just want to check this is intentionally left in and not debugging logging

Reply:
Not intentional. I'll remove it.